src/explorer/SiteTreeItem.ts

```diff
@@ -124,11 +123,25 @@ export abstract class SiteTreeItem implements IAzureParentTreeItem {
             }
         }
         cancelWebsiteValidation(this);
+        // tslint:disable-next-line:strict-boolean-expressions
+        if (workspace.workspaceFolders && workspace.workspaceFolders.length === 1) {
+            const currentWorkspace: WorkspaceFolder = workspace.workspaceFolders[0];
+            if (currentWorkspace.uri.fsPath === fsPath || this.isSubpath(currentWorkspace.uri.fsPath, fsPath)) {
```

Comment:
You can't compare paths directly (it won't work in several edge cases - like `\example\path` and `\example\..\example\path`). You have to do something like this: https://github.com/Microsoft/vscode-azurefunctions/blob/master/src/utils/fs.ts#L92